nservicebus/pipeline/customizing-v6.md

```diff
@@ -50,6 +50,8 @@ The following lists describe some of the common stages which you can build your
 
 ### Outgoing Pipeline Stages
 * Operation specific processing: There is a dedicated stage for each bus operation (e.g. Send, Publish, Subscribe, ...). Behaviors can use one of the following contexts: `IOutgoingSendContext`, `IOutgoingPublishContext`, `IOutgoingReplyContext`, `ISubscribeContext`, `IUnsubscribeContext`.
+* Audit message processing: In here behaviors have access to audited message and audit address. Behaviors should use `IAuditContext` to enlist in this stage.
+* Fault message processing: This is a dedicated stage for processing faulty messages. Behaviors have access to message, exception and error address. This stage provides `IFaultContext` to it's behaviors.  
```

Comment:
What about:
"This is a dedicated stage for processing faulty messages that are being moved to the error queue" ?